features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/NewOverview/NewCustomizedStepper.jsx

```diff
@@ -0,0 +1,427 @@
+                                </Tooltip>
+                            )}
+                            {label === 'Publish' && (
+                                <div>
+                                    {lifecycleState !== 'Published' ? (
+                                        <Button
+                                            size='small'
+                                            variant='contained'
+                                            color='primary'
+                                            onClick={() => updateLCStateOfAPI(api.id, 'Publish')}
+                                            disabled={((api.type !== 'WEBSUB' && !isEndpointAvailable))
+                                                || !deploymentsAvailable
+                                                || !isTierAvailable
+                                                || api.isRevision || AuthManager.isNotPublisher()
+                                                || api.workflowStatus === 'CREATED'}
+                                        >
+                                            Publish
+                                            {isUpdating && <CircularProgress size={20} />}
+                                        </Button>
+                                    ) : (
+                                        <Grid
+                                            container
+                                            direction='row'
+                                            alignItems='center'
+                                            justify='center'
+                                        >
+                                            <Grid item>
+                                                <CheckIcon className={classes.iconTrue} />
+                                            </Grid>
+                                            <Grid item style={{ marginBottom: '3px', marginLeft: '3px' }}>
+                                                <Typography variant='h7'>
+                                                    <FormattedMessage
+                                                        id='Apis.Details.Overview.CustomizedStepper.publish'
+                                                        defaultMessage=' Published (Current API)'
+                                                    />
+                                                </Typography>
+                                            </Grid>
+                                        </Grid>
+                                    )}
+                                </div>
+                            )}
+                        </StepLabel>
+                    </Step>
+                ))}
+            </Stepper>
+        </div>
+    );
+}
+
+NewCustomizedStepper.propTypes = {
+    classes: PropTypes.shape({}).isRequired,
+    api: PropTypes.shape({
+        id: PropTypes.string,
+    }).isRequired,
+};
+
+NewCustomizedStepper.contextType = ApiContext;
```

Comment:
`contextType` is only valid for class components? 

This is a no-op actually 😀 

https://reactjs.org/docs/context.html#classcontexttype

Reply:
Fixed